src/main/java/com/binance/api/client/impl/BinanceApiAsyncRestClientImpl.java

```diff
@@ -89,11 +89,21 @@ public void getCandlestickBars(String symbol, CandlestickInterval interval, Bina
   public void get24HrPriceStatistics(String symbol, BinanceApiCallback<TickerStatistics> callback) {
     binanceApiService.get24HrPriceStatistics(symbol).enqueue(new BinanceApiCallbackAdapter<>(callback));
   }
+  
+  @Override
+  public void getAll24HrPriceStatistics(BinanceApiCallback<List<TickerStatistics>> callback) {
+	  binanceApiService.getAll24HrPriceStatistics().enqueue(new BinanceApiCallbackAdapter<>(callback));
```

Comment:
Please use 2 whitespaces for indentation.

Reply:
Done, But I would have much preferred tab for indentation. Its more standard In my opinion :)